app/index.js

```diff
@@ -103,11 +76,18 @@ var onError = function(error){
   }
   process.exit(-1);
 };
-var onListening = function(){
-  var listenurl = (nconf.get('https')?'https':'http:')+'://'+nconf.get('listen')+':'+nconf.get('port');
+function onListening() {
+  let listenurl = (nconf.get('https')?'https':'http:')+'://'+nconf.get('listen')+':'+nconf.get('port');
   console.log('OpenTMI started on ' +listenurl+ ' in '+ nconf.get('cfg')+ ' mode');
+  eventBus.emit('start_listening', {url: listenurl});
 };
 
 server.listen(nconf.get('port'), nconf.get('listen'));
 server.on('error', onError);
 server.on('listening', onListening);
+
+// this would be usefull for testing
+module.exports = {
```

Comment:
could use object naming shortcuts and make it a one liner `module.exports = { server, eventBus }`